### DISCUSSION TITLE
rendered.h / renderer.cpp file location

### DISCUSSION THREAD

**Discussion by jeroenvermeulen** [arendst/Tasmota#10823] [Ideas]
@gemu2015 Would it maybe be a good idea to move the `renderer.h` and `renderer.cpp` to a more generic location?
I think it is kind of messy it is inside the `esp-epaper-29-ws-20171230-gemu-1.1/src` dir but it is used in a growing number of drivers. 
I am working on one extra driver.
Maybe I can send a PR to move the Renderer files?

**Comment by gemu2015** [comment]
@jeroenvermeulen 

Yes this would be a good idea to split this folder into 3.  (renderer, epaper29,epaper42) !
No problems, of course you can.
